### PR TITLE
OEL-1910: Drupal 9.4 support.

### PR DESCRIPTION
**Pull request by drishu**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,7 +25,7 @@
         "drupal/facets_form": "1.0.0-alpha3",
         "drupal/field_group": "^3.2",
         "drupal/file_link": "^2.0.6",
-        "drupal/honeypot": "^2.0",
+        "drupal/honeypot": "^2.0.2",
```
Comment (drishu) [inline composer.json:28]: this specific version is a oe_contact_forms requirement

File: composer.json [hunk 2]
```diff
@@ -13,7 +13,8 @@
         "drupal/better_exposed_filters": "^5.0",
         "drupal/block_field": "^1.0@RC",
         "drupal/composite_reference": "^2.1",
-        "drupal/core-recommended": "~9.3.20",
+        "drupal/core-recommended": "~9.4.0",
+        "drupal/ctools": "^3.11",
```
Comment (drishu) [inline composer.json:17]: this fixes a major bug, see the readme

File: composer.json [hunk 3]
```diff
@@ -40,31 +41,29 @@
...
+        "openeuropa/oe_contact_forms": "^1.5",
         "openeuropa/oe_content": "^3.0.0-alpha4",
-        "openeuropa/oe_content_extra": "^1.0@beta",
-        "openeuropa/oe_corporate_blocks": "^4.4",
+        "openeuropa/oe_content_extra": "^1.0.0-beta2",
+        "openeuropa/oe_corporate_blocks": "^4.8",
         "openeuropa/oe_corporate_countries": "^2.0.0-alpha6",
-        "openeuropa/oe_dashboard_agent": "^0.4.0",
-        "openeuropa/oe_list_pages": "^0.19",
-        "openeuropa/oe_media": "^1.14",
```
Comment (drishu) [inline composer.json:51]: I removed oe_media as its a dependency of oe_content (so its still in the lock), maybe I should have left the explicit require, not sure, but its not listed as a direct req of our modules

**Review by drishu — COMMENTED** [review]
rest of them are 9.4 comp bumps of our modules

File: composer.json [hunk 4]
```diff
@@ -116,14 +115,8 @@
         "installer-types": ["npm-asset", "bower-asset"],
         "patches": {
             "drupal/core": {
-                "Allow altering the post_update run order @see https://www.drupal.org/project/drupal/issues/3129231": "https://www.drupal.org/files/issues/2021-10-28/31292311-12.patch",
+                "Allow altering the post_update run order @see https://www.drupal.org/project/drupal/issues/3129231": "https://www.drupal.org/files/issues/2022-06-27/drupal-post_update_alter-3129231-20.patch",
                 "Incorrect label in taxonomy glossary for terms that start with zero @see https://www.drupal.org/project/drupal/issues/3291520": "patches/3291520-9.3.x.patch"
```
Comment (brummbar) [inline composer.json]: This we can drop it and use https://www.drupal.org/files/issues/2022-08-01/3291520-12.patch

**Review by brummbar — CHANGES_REQUESTED** [review]

**Review by brummbar — APPROVED** [review]

**Pull request merged**